app/models/application_record.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
```

Comment:
nice. Remember someone pitching this before (jose valím?) but it was shot down by rails core.
I like it a lot